kin-devplatform-sdk/src/main/java/kin/devplatform/exception/BlockchainException.java

```diff
@@ -8,7 +8,7 @@
 
 	@IntDef({ACCOUNT_CREATION_FAILED, ACCOUNT_NOT_FOUND,
 		ACCOUNT_ACTIVATION_FAILED, INSUFFICIENT_KIN,
-		TRANSACTION_FAILED, ACCOUNT_LOADING_FAILED, ACCOUNT_CREATION_TIMEOUT, UNKNOWN})
+		TRANSACTION_FAILED, ACCOUNT_LOADING_FAILED, ACCOUNT_CREATION_TIMEOUT, BLOCKCHAIN_VERSIONS_ARE_NOT_THE_SAME, UNKNOWN})
```

Comment:
I would change the `BLOCKCHAIN_VERSIONS_ARE_NOT_THE_SAME` to migration needed as well, it's way clearer

Reply:
ok